processing/src/main/java/org/apache/druid/error/InvalidInput.java

```diff
@@ -35,7 +35,7 @@ public static DruidException exception(Throwable t, String msg, Object... args)
   private final String msg;
   private final Object[] args;
 
-  public InvalidInput(
+  protected InvalidInput(
```

Comment:
Why this change?  Please back it out.  `protected` pretty much doesn't help anybody, might as well be public.

Reply:
Changed this because we had a couple of static InvalidInput exception creators that mimic this signature `exception(...)`, therefore any piece of code that intends to call this constructor can use the creator instead for the time being.

Reverted back the change in my newer commit.